data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua

```diff
@@ -39,7 +39,8 @@ obj_initial_close_objectives_window = {
    body = objective_text(_("Handling of the ‘Objectives’ window"),
       li_image("images/wui/menus/objectives.png", _([[You can open and close the ‘Objectives’ window by clicking on the ‘Objectives’ button in the toolbar on the bottom of the screen.]])) ..
       li_arrow(_([[Like most other windows, you can also close the window by right-clicking on it.]])) ..
-      li_arrow(_([[When you have accomplished an objective, it will disappear from the list.]]))
+      li_arrow(_([[When you have accomplished an objective, it will disappear from the list.]])) ..
+      li_arrow(_([[You may open the objectives window now.]]))
```

Comment:
you may leave out this item as it is automatically opened by  line 48 in the mission_thread

Reply:
> you may leave out this item as it is automatically opened by line 48 in the mission_thread

Oh, *that's* what's happening... so... not really a bug then like I reported. A bit confusing but I get what's happening now.